src/handlers/counting/repost/reposting.ts

```diff
@@ -1,5 +1,12 @@
-import type { Message } from "discord.js";
+import type { DiscordAPIError, Message } from "discord.js";
+import { countingLogger } from "../../../utils/logger/counting";
+import { inspect } from "util";
 
 export default function repostWithReposting(message: Message<true>): Promise<Message> {
-  return message.channel.send(`${message.author.toString()}: ${message.content}`).catch(() => message);
+  return message.channel.send(`${message.author.toString()}: ${message.content}`)
+    .catch((err: DiscordAPIError) => {
```

Comment:
`err` is unknown, not necessarily a `DiscordAPIError`. We're handling with the inspect util so we don't need to force this type